manifests/kubernetes/deployment.yaml

```diff
@@ -121,7 +121,7 @@ spec:
           type: Directory
       - name: proc
         hostPath:
-          path: /proc
```

Comment:
I suppose this change breaks our CI on kind for now.

Reply:
btw, the integration CI on kind timed out twice. Need some investigation.